packages/manager/modules/enterprise-cloud-database/src/create/choose-commitment-period/choose-commitment-period.html

```diff
@@ -14,7 +14,8 @@
             <div class="mb-2 font-weight-bold">
                 <ovh-manager-enterprise-cloud-database-price-component
                     data-price="$ctrl.enterpriceDb.cluster.price.price * ($ctrl.enterpriceDb.defaultReplicaCount + $ctrl.enterpriceDb.additionalReplica.value) * commitmentPeriod"
-                    data-tax="$ctrl.enterpriceDb.cluster.price.tax * ($ctrl.enterpriceDb.defaultReplicaCount + $ctrl.enterpriceDb.additionalReplica.value) * commitmentPeriod">
+                    data-tax="$ctrl.enterpriceDb.cluster.price.tax * ($ctrl.enterpriceDb.defaultReplicaCount + $ctrl.enterpriceDb.additionalReplica.value) * commitmentPeriod"
```

Comment:
`enterprice` -> `enterprise`
Also I think we should avoid calculation in the views. Maybe this could be extracted to a service as it could be reused